
---
sensor_msgs/PointCloud2 pointcloud
sensor_msgs/Image image